src/Microsoft.DotNet.XHarness.Android/Execution/AdbProcessManager.cs

```diff
@@ -34,8 +39,9 @@ public ProcessExecutionResults Run(string adbExePath, string arguments, TimeSpan
             RedirectStandardOutput = true,
             RedirectStandardError = true,
             FileName = adbExePath,
-            Arguments = $"{deviceSerialArgs} {arguments}",
+            Arguments = processArgs,
```

Comment:
Now that you have arguments in a list you should assign it to `ArgumentList` instead of `Arguments`. This will take care of escaping them properly.

Reply:
Ah, right. I need to then switch the Xamarin code to use `ArgumentList` too (need to talk to them first) because we could remove a lot of code then.